src/proxyClient.js

```diff
@@ -223,6 +223,14 @@ worker.onmessage = function worker_onmessage(event) {
       }
       break;
     }
+    case 'custom': {
+      if (Module.onCustomMessage) {
```

Comment:
these should be `Module['onCustomMessage']`, as closure compiler might otherwise minify them into `Module.ax` or some other minified name.